README.md

```diff
@@ -42,6 +42,8 @@ cd stacks-core
 ### 3. Build the project
 
 ```bash
+# Make sure you have necessary build tools installed ie. GCC or Clang
+sudo apt-get install build-essential
```

Comment:
If the goal here is an os specific example for installing dependencies/building, it should be in a dedicated debian README in the docs folder. However, that would also predicate on having similar readme's for other OS (i.e. windows/mac/rpm based dist etc)